mesheryctl/internal/cli/root/system/channel_test.go

```diff
@@ -105,20 +105,11 @@ func TestRunChannelWithNoCmdOrFlag(t *testing.T) {
 	SetupFunc(t)
 	SystemCmd.SetArgs([]string{"channel"})
 	err = SystemCmd.Execute()
-	if err != nil {
-		t.Error(err)
-	}
-
-	currCtx, err := mctlCfg.GetCurrentContext()
 
-	if err != nil {
-		t.Error(err)
-	}
+	actualResponse := err.Error()
 
-	actualResponse := b.String()
-	expectedResponse := ""
-	expectedResponse += PrintChannelAndVersionToStdout(*currCtx, "local") + "\n\n"
-	expectedResponse += channelCmd.UsageString()
+	expectedResponse := "please specify a flag or subcommand. Use 'mesheryctl system channel --help' to display user guide." + "\n\n"
+	expectedResponse += "See https://docs.meshery.io/reference/mesheryctl/system/channel for usage details\n"
```

Comment:
For error cases just asserting an error code/response should be fine.

Not sure if there is a need for string matching for this

Reply:
I don't change the error code.

Yes modify error message. So to confirm that the message I expect is same, I match error strings.

Reply:
I don't mean to say you do. What I mean to say just a check on error code should be good enough